packages/core/src/components/WarningPanel/WarningPanel.test.tsx

```diff
@@ -15,23 +15,61 @@
  */
 
 import React from 'react';
+import { fireEvent } from '@testing-library/react';
 import { renderInTestApp } from '@backstage/test-utils';
+import { Typography } from '@material-ui/core';
 
 import { WarningPanel } from './WarningPanel';
 
-const minProps = { title: 'Mock title', message: 'Some more info' };
+const propsTitle = { title: 'Mock title' };
+const propsTitleMessage = { title: 'Mock title', message: 'Some more info' };
+const propsMessage = { message: 'Some more info' };
 
 describe('<WarningPanel />', () => {
   it('renders without exploding', async () => {
-    const { getByText } = await renderInTestApp(<WarningPanel {...minProps} />);
-    expect(getByText('Mock title')).toBeInTheDocument();
+    const { getByText } = await renderInTestApp(
+      <WarningPanel {...propsTitleMessage} />,
+    );
+    expect(getByText('Warning: Mock title')).toBeInTheDocument();
```

Comment:
Purely informational - did you know of `screen`? I personally get bored fast with using the output of the render function (and particularly to deconstruct it, even though that of course is optional), and learned that the below seems to be the recommended way these days ... and i like it!

```ts
import { screen } from '@testing-library/react';
await renderInTestApp(<WarningPanel {...propsTitleMessage} />);
expect(screen.getByText('Warning: Mock title')).toBeInTheDocument();
```

it's always there :) and even if you re-render or make multiple separate renders etc, it just works. Maybe you'll like it too.

Reply:
I didn't!  So basically, avoiding creating the new getByText object by using the built in `screen`?  very nice.

for these, i sort of just copied what was there and expanded, but let me play a bit.

Reply:
Entirely optional - it's a matter of taste but Dodds himself started recommending doing things like that so I went with it :)

Reply:
Gave it a shot. It's for sure easier to read on the eyes.

"Dodds himself"... Kent C Dodds is it?  New name to me.  In my part of the world, if you said Gene, Nicole, Jezz, etc., folks in the DevOps space I'd be right there with you. I might pick up on a Gosling or Torvalds quote, as well, but need to lookup Dodds now. 😁 My firm doesn't do a lot of Javascript, and even less (next to no) React so I'm diving in to learn!

Reply:
Haha, understandable. He wrote the testing library in question.